doc/modules/tablereport/altering_appearance.rst

```diff
@@ -0,0 +1,22 @@
+.. |TableReport| replace:: :class:`~skrub.TableReport`
```

Comment:
All of the tablereport pages are very short I think they could have stayed in one page to avoid one level of nesting. there is not so much to say about the table report that it requires multiple web pages and this way we shorten the path `home > user guide TOC > tablereport TOC > tablereport` to reach the first line of actual content about the table report